mozetl/taar/taar_similarity.py

```diff
@@ -134,16 +178,28 @@ def get_donor_pools(users_df, clusters_df, num_donors, random_seed=None):
     # Get the specific number of donors for each cluster and drop the
     # predicted cluster number information.
     current_sample_size = donor_df.count()
+
+    sample_pcnt = float(num_donors) / current_sample_size
+    # TODO: Adjust the sample_size computation so that the 
+    # value of num_donors is scaled such that the sample percentage
+    # approaches 0.12 so that we get ~1000 rows in donor_pool_df
```

Comment:
@mlopatka when I ran this with the WHITELIST.BASIC whitelist - that is 60 days and 3 stars or more - we get 1000 donors with the 0.12 scaling factor.